posthog/queries/stickiness/__init__.py

```diff
@@ -1 +1,8 @@
-from .stickiness import *
+from posthog.settings import EE_AVAILABLE
+
+if EE_AVAILABLE:
+    from ee.clickhouse.queries.stickiness import ClickhouseStickiness as Stickiness
```

Comment:
Query classes are imported all over the place such as the cohort recalculation area. When ee is available, it should utilize the enterprise version of queryclass. This is the pattern that can allow that sort of import overriding. I do think it's an antipattern that could use some rethinking because it's not exactly clear from a developer standpoint which class you're dealing with